### DISCUSSION TITLE
mpeg2 TS fail

### DISCUSSION THREAD

**Discussion by prompter21** [OvenMediaLabs/OvenMediaEngine#1107] [Help]
I am using OME on ubuntu 20.04. When I log in to the server with UDP TS signals and play back with HLS, DASH, or LLHLS, everything goes fine, but after a few hours the playback goes into a repetitive loop and doesn't come out of there until I restart the login or restart the OME service. .
I use various versions of OME, from 12.1 to 14.5.

**Comment by getroot** [comment]
Please fill in the detailed information requested by the form.

**Comment by stevespaw** [comment]
I have had the same issue....
